addons/packages/load-balancer-and-ingress-service/1.6.1/bundle/config/upstream/loadbalancerandingressservice/ais.yaml

```diff
@@ -0,0 +1,11 @@
+---
```

Comment:
This should not be in the base template, because when the user doesn't specify any control plane network, you should not create a AIS.